### PR TITLE
Bugfix/18210 hca form prefill msgs

### PR DESCRIPTION
**Pull request by tlei123**
## Description
18210 - Add missing prefill-messages to HCA form-pages that are prefillable -- see **[Prefillable pages](#issuecomment-505659671)** list in Comment below.

## Testing done
Local.  All HCA unit-tests still passing.

## Screenshots
See original issue [17861](/department-of-veterans-affairs/vets.gov-team/issues/17861), where I only added prefill-message to the screen-captured page.  This new PR is for all the other pages that are missing prefill-messages.

## Acceptance criteria
- [ ] Prefill-messages show on all prefillable pages.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/hca/config/form.js [hunk 1]
```diff
@@ -508,7 +510,12 @@ const formConfig = {
           title: 'Service history',
           uiSchema: {
             'ui:title': 'Service history',
-            'ui:description': 'Check all that apply to you.',
+            'ui:description': !environment.isProduction()
```
Comment (jbalboni) [inline src/applications/hca/config/form.js]: Do you need this flag? None of the other changes appear to be behind a flag.

File: src/applications/hca/config/form.js [hunk 2]
```diff
@@ -642,7 +654,8 @@ const formConfig = {
           title: 'Financial disclosure',
           uiSchema: {
             'ui:title': 'Financial disclosure',
-            'ui:description': financialDisclosureText,
+            'ui:description': PrefillMessage,
```
Comment (jbalboni) [inline src/applications/hca/config/form.js]: I don't think there's any prefilled information on this page, it's just the yes/no field and a warning.

File: src/applications/hca/config/form.js [hunk 3]
```diff
@@ -674,8 +691,11 @@ const formConfig = {
               formData.maritalStatus.toLowerCase() === 'separated'),
           uiSchema: {
             'ui:title': 'Spouse’s information',
-            'ui:description':
-              'Please fill this out to the best of your knowledge. The more accurate your responses, the faster we can process your application.',
+            'ui:description': PrefillMessage,
```
Comment (jbalboni) [inline src/applications/hca/config/form.js]: I don't think there's any prefilled info on this page. We don't have anywhere to get spouse info from.

File: src/applications/hca/config/form.js [hunk 4]
```diff
@@ -508,7 +510,12 @@ const formConfig = {
           title: 'Service history',
           uiSchema: {
             'ui:title': 'Service history',
-            'ui:description': 'Check all that apply to you.',
+            'ui:description': !environment.isProduction()
+              ? MilitaryPrefillMessage
+              : undefined,
+            'view:textObject': {
```
Comment (tlei123) [inline src/applications/hca/config/form.js]: @jbalboni , for adding prefill-message to pages that already have 'ui:description' prop, unlike the 2 solution-ideas we discussed yesterday, I found a much easier, cleaner way -- 'view:textObject' prop.

**Review by jbalboni — COMMENTED** [review]
@tzelei Started making comments about which pages should have the message, but it's probably better for you to go through the fields in https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/form_profile_mappings/1010ez.yml and remove the message from the pages that don't have any of the listed fields.

File: src/applications/hca/config/form.js (hunk 2 above)
Comment (tlei123) [reply src/applications/hca/config/form.js]: Prefill message removed.

File: src/applications/hca/config/form.js (hunk 3 above)
Comment (tlei123) [reply src/applications/hca/config/form.js]: Prefill-message removed.

File: src/applications/hca/config/form.js (hunk 1 above)
Comment (tlei123) [reply src/applications/hca/config/form.js]: > Do you need this flag? None of the other changes appear to be behind a flag.

@erikphansen , should I remove all conditional flags like the one above for the MilitaryPrefillMessage instances?  I don‘t know when the relevant epic/phase this belongs to (HCA update “phase 2”?).

**Comment by tlei123** [comment]
More changes pushed up now -- unnecessary prefill-messages removed.  Thx, @jbalboni for clarifiying which pages need prefill-messages!

Prefillable pages now enumerated in ticket description, and copied here...

### Prefillable pages
ONLY the following form-pages are prefillable and need prefill-messages, according to the [vets-api 1010ez form-profile-mapping](/department-of-veterans-affairs/vets-api/blob/master/config/form_profile_mappings/1010ez.yml):
- veteranInformation (chapter):
  - veteranInformation
  - birthInformation
  - demographicInformation
  - veteranAddress
  - contactInformation
- militaryService:
  - serviceInformation
  - additionalInformation
- vaBenefits:
  - vaBenefits

All other pages do NOT need prefill-messages.

**Review by jbalboni — APPROVED** [review]

File: src/applications/hca/config/form.js (hunk 1 above)
Comment (tlei123) [reply src/applications/hca/config/form.js]: Flags removed.

**Pull request merged**